plugins/registry.json

```diff
@@ -34,6 +34,15 @@
         "title": "Suggest stuff.",
         "icon_url": "https://i.imgur.com/qtE7AH8.png",
         "thumbnail_url": "https://i.imgur.com/qtE7AH8.png"
+    },
+	"reminder": {
```

Comment:
Indentation error here. The name of the plugin should be indented only once (not twice)

Reply:
![image](https://user-images.githubusercontent.com/55140357/224558335-7b6c07dd-a607-4b40-93c4-d47d47639636.png)
Looks fine for me on my end hmm

Reply:
Also tried a different program, its showing like this.
Do you have an idea wihy its wrong on github?
![image](https://user-images.githubusercontent.com/55140357/224558766-1ea130ab-528e-4ef6-8049-3b3863352ac2.png)

Reply:
Yeah it looks fine in VSC, (incl. the GH hosted VSC) not sure why it appears weird on the normal instance